.github/workflows/main.yml

```diff
@@ -52,10 +58,12 @@ jobs:
               run: make install
 
             - name: Run phpcs
-              run: make phpcs-with-cache
+              if: steps.changed-files.outputs.any_changed == 'true'
+              run: ./vendor/bin/phpcs -d memory_limit=-1 --cache=phpcs.cache --standard=./vendor/spryker/code-sniffer/Spryker/ruleset.xml --parallel=75 ${{ steps.changed-files.outputs.all_changed_files }}
 
             - name: Run codeception
               run: make codeception-without-coverage
 
             - name: Run phpstan
-              run: make phpstan
+              if: steps.changed-files.outputs.any_changed == 'true'
+              run: ./vendor/bin/phpstan --memory-limit=-1 analyse ${{ steps.changed-files.outputs.all_changed_files }}
```

Comment:
Does this work with multiple file path like this? Did you test this locally?

Reply:
I tested it with 2 changed php Files and it worked perfektly